.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature request
+description: Suggest a new idea for FastTrackML
+title: "[FEATURE REQUEST]: <title>"
+labels: [ "feature" ]
+
+body:
+  - type: textarea
+    id: related-problem
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+    validations:
+      required: true
```

Comment:
Feature request may not always relate to a "problem" -- "I wish this software had feature X" -- maybe shouldn't be required.

Reply:
Made this optional and prioritized the feature description instead